modules/nw-ipfailover-configuration.adoc

```diff
@@ -41,7 +41,51 @@ $ oc adm policy add-scc-to-user privileged -z ipfailover
 ----
 $ oc adm policy add-scc-to-user hostnetwork -z ipfailover
 ----
+
+. {rh-openstack-first} only: Complete the following steps to make a failover VIP address reachable on {rh-openstack} ports.
++
+.. Use the {rh-openstack} CLI to show the default {rh-openstack} API and VIP addresses in the `allowed_address_pairs` parameter of your {rh-openstack} cluster:
++
+[source,terminal]
+----
+$ openstack port show <cluster_name> -c allowed_address_pairs
+----
++
+.Output example
+[source,terminal,subs="+attributes"]
+----
+*Field*                  *Value*
+allowed_address_pairs    ip_address='192.168.0.5', mac_address='fa:16:3e:31:f9:cb'
+                         ip_address='192.168.0.7', mac_address='fa:16:3e:31:f9:cb'
+----
++
+.. Set a different VIP address for the IP failover deployment and make the address reachable on {rh-openstack} ports by entering the following command in the {rh-openstack} CLI. Do not set any default {rh-openstack} API and VIP addresses as the failover VIP address for the IP failover deployment.
++
+.Example of adding the `1.1.1.1` failover IP address as an allowed address on {rh-openstack} ports.
+[source,terminal,subs="+attributes"]
+----
+$ openstack port set <cluster_name> --allowed-address ip-address=1.1.1.1,mac-address=fa:fa:16:3e:31:f9:cb
+----
++
+.. Create a deployment YAML file to configure IP failover for your deployment. See "Example deployment YAML for IP failover configuration" in a later step.
 +
+.. Specify the following specification in the IP failover deployment so that you pass the failover VIP address to the `OPENSHIFT_HA_VIRTUAL_IPS` environment variable:
++
+.Example of adding the `1.1.1.1` VIP address to `OPENSHIFT_HA_VIRTUAL_IPS`
+[source,yaml,subs="attributes,quotes"]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ipfailover-keepalived
+# ...
+      spec:
+          env:
+          - name: OPENSHIFT_HA_VIRTUAL_IPS 
+          value: "1.1.1.1"
+# ...
+----
+
 . Create a deployment YAML file to configure IP failover:
```

Comment:
Do you need to mention that this step is not needed for Red Hat OpenStack Platform (RHOSP)? Is there a way to ifdef these two steps, 3 and 4, to avoid the steps that are not needed per flavor?